### ISSUE TITLE
Help Button is not directing to right URL in properties menu of circuit element.

### ISSUE THREAD

**Issue by aashimawadhwa** [CircuitVerse/CircuitVerse#3002]
**Describe the bug**
help button is not directing to right URL in properties menu of  circuit element in simulator.

**To Reproduce**
1. Select an circuit element
2. Click on help from properties menu
3.  See error

**Expected behavior**
It should be directing to circuitverse docs

**Screenshots**
![Screenshot from 2022-03-05 01-26-52](https://user-images.githubusercontent.com/73706697/156833014-70af6e76-86ee-4d19-9748-d479854733a5.png)

![Screenshot from 2022-03-05 01-26-58](https://user-images.githubusercontent.com/73706697/156833021-c2c76e18-68be-4120-9db2-6a8cc7499fa3.png)

**Are you working on this issue? (Yes/No)**
**yes**

**Comment by vedant-jain03** [comment]
@CircuitVerse/circuitverse-core-team What should be the ideal path for help link?

**Comment by kushalShukla-web** [comment]
Can I contribute to it

**Comment by pavanjoshi914** [comment]
circuitverse docs are in the process of refactoring. if refactoring is finished you can start to change the links 

you can see the difference, so each  doc is present in particular ```chapter``` folder and links are different now
```
https://docs.circuitverse.org/#/chapter4/3output?id=output
```

**Comment by kushalShukla-web** [comment]
@pavanjoshi914  is this issue assigned to me?

**Comment by aashimawadhwa** [comment]
> circuitverse docs are in the process of refactoring. if refactoring is finished you can start to change the links
> 
> you can see the difference, so each doc is present in particular `chapter` folder and links are different now
> 
> ```
> https://docs.circuitverse.org/#/chapter4/3output?id=output
> ```

should I wait for the refactoring to be done?

**Comment by pavanjoshi914** [comment]
> @pavanjoshi914 is this issue assigned to me?

@kushalShukla-web I think  @aashimawadhwa  already mentioned working on this issue
```
Are you working on this issue? (Yes/No)
yes
```

I think a great place to start is going with issues labelled ```good first issue```

if someone is not working on that issue, or if issue is pending for a long time you can start working on such issue without waiting for issue being assigned

hope this helps!

**Comment by pavanjoshi914** [comment]
> > circuitverse docs are in the process of refactoring. if refactoring is finished you can start to change the links
> > you can see the difference, so each doc is present in particular `chapter` folder and links are different now
> > ```
> > https://docs.circuitverse.org/#/chapter4/3output?id=output
> > ```
> 
> should I wait for the refactoring to be done?

cc @Shivansh2407

**Comment by tachyons** [comment]
> circuitverse docs are in the process of refactoring. if refactoring is finished you can start to change the links

Already deployed updated docs, so we can start fixing links

**Comment by pavanjoshi914** [comment]
@aashimawadhwa  you can start working on this

**Comment by Prerna-0202** [comment]
Fixed in #3215
